english/marshmallow.md

```diff
@@ -1,5 +1,7 @@
 There's a spot open for the role of [Lead Programmer](programmer/lead_programmer.md)
 
+There's a bow in your hand, do you [shoot the target](shoot-target/shoot-target.md) or [shoot the sky](shoot-air/shoot-air.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.